.github/workflows/autopublish.yml

```diff
@@ -0,0 +1,56 @@
+name: Faceted Search Auto Publish
+
+on:
+  push:
+    branches: [master, develop, main]
+
+jobs:
+  build_and_publish:
+    name: Build and publish
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js 12
+        uses: actions/setup-node@v1
+        with:
+          node-version: 12
+          registry-url: 'https://registry.npmjs.org/'
+
+      - name: Get package.json version
+        id: current-version
+        run: |
+          localVersion="$(cat package.json | jq -r ".version")"
+          echo "Local version is $localVersion"
+          echo "::set-output name=current_version::$localVersion"
+
+      - name: Check if package.json version is already published
+        id: check-already-published
+        run: |
+          alreadyPublished="$(yarn info @talend/react-faceted-search versions | sed -n "/'${{ steps.current-version.outputs.current_version }}'/ p" | wc -l)"
+          echo "Is local version already published ? $alreadyPublished"
+          echo "::set-output name=already-published::$alreadyPublished"
+
+      - name: Install
+        if: ${{ steps.check-already-published.outputs.already-published == 0 }}
+        run: yarn install --frozen-lockfile
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.NPM_AUTH_TOKEN }}
```

Comment:
I think we don't need the NPM token for that as we are not supposed to fetch private dependencies

Reply:
locale package is private, we'll fix it in the next release